### DISCUSSION TITLE
Errors by project: XXXXX-> 'API responded with status 429'\"

### DISCUSSION THREAD

**Discussion by rsalvagio** [airbytehq/airbyte#29082] [Connector Questions]
How to limit the requests to set up a Jira connection?

bash <(curl -s https://raw.githubusercontent.com/faros-ai/airbyte-local-cli/main/airbyte-local.sh) \
--src 'farosai/airbyte-faros-feeds-source' \
--src.feed_cfg.feed_name 'jira-feed' \
--src.feed_cfg.email 'aaaaa@bbbbb.de' \
--src.feed_cfg.api_token "xxxxxxxxxxxxxxxxxxxxxx" \
--src.feed_cfg.domain 'https://xxxxxxx/jira/' \
--src.feed_cfg.projects_query_mode.query_mode 'ProjectList' \
--src.feed_cfg.projects '["XXXXX"]' \
--src.feed_cfg.reject_unauthorized false \
--dst.edition_configs.hasura_url 'http://host.docker.internal:8080/'  \
--dst.edition_configs.hasura_admin_secret 'XXXXX' \
--dst.edition_configs.edition 'community' \
--dst.edition_configs.verify_ssl_cert false \
--dst 'farosai/airbyte-faros-destination' \
--src.feed_cfg.concurrency_limit 1 \
--no-src-pull \
--no-dst-pull \
--max-cpus "1"  2>&1 | tee log.txt

**Comment by marcosmarxm** [comment]
Closing because the question is related to Faros CLI.
